container/state.go

```diff
@@ -68,6 +69,10 @@ func NewState() *State {
 
 // String returns a human-readable description of the state
 func (s *State) String() string {
+	if s.CommitInProgress > 0 {
+		return "Commit In Progress"
+	}
+
```

Comment:
I think we could add a method like `IsCommitting` to check if this container being commit.